packages/remark-shiki-twoslash/src/index.ts

```diff
@@ -7,11 +7,21 @@ import visit from "unist-util-visit"
 import { addIncludes, replaceIncludesInCode } from "./includes"
 import { cachedTwoslashCall } from "./caching"
 
+export type Settings = UserConfigSettings & {
+  wrapFragments?: true
+}
```

Comment:
I think this should probably be added in `shiki-twoslash` so that all the config types are in one place ( even though I totally see the value in having it here)